test/lexer_test.dart

```diff
@@ -6,14 +6,14 @@ import 'package:test/test.dart' as test;
 
 void main() {
```

Comment:
Add warnings test. We only have a single warning for now which is extra semicolons, so test that.